README.md

```diff
@@ -55,7 +55,8 @@ git clone https://github.com/stripe-samples/oxxo-payment
 Copy the .env.example file into a file named .env in the folder of the server you want to use. For example:
 
 ```
-cp .env.example server/node/.env
+cd oxxo-payment
+cp .env.example server/<node|php|python|ruby>/.env 
```

Comment:
If you want to include all the options here instead of just an example, let's remove "For example:" on line 55